conans/client/graph/graph.py

```diff
@@ -284,7 +284,7 @@ def new_build_order(self):
             new_level = []
             for n in level:
                 if n.binary == BINARY_BUILD and n.pref not in total_prefs:
-                    new_level.append((n.id, repr(n.pref)))
+                    new_level.append((n.id, n.pref.copy_clear_rev()))
```

Comment:
`copy_clear_revs()` instead? why you want to keep the `rrev` and not the `prev`?

Reply:
No, we certainly want to build a revision, the build order will be building recipes with a given revision, that shouldn't change.